packages/elements/src/radio-button/__test__/radio-button.test.js

```diff
@@ -447,6 +447,21 @@ describe('radio-button/RadioButton', () => {
       const checkedRadio = group.find(element => element.checked);
       expect(checkedRadio.id).to.equal('btn2');
     });
+    it('Should separate scope between shadow dom and light dom', async () => {
+      const radio = await fixture('<ef-radio-button name="group" id="btn1" checked>1</ef-radio-button>');
+      const container = await fixture('<div></div>');
+      container.attachShadow({ mode: 'open' });
+      const radio2 = document.createElement('ef-radio-button');
+      radio2.setAttribute('name', 'group');
+      radio2.setAttribute('id', 'btn2');
+      radio2.checked = true;
+      container.shadowRoot.appendChild(radio2);
+      await elementUpdated(radio);
+      await elementUpdated(radio2);
+
+      expect(radio.checked).to.equal(true);
+      expect(radio2.checked).to.equal(true);
+    });
```

Comment:
A better test case would be to create HTML like below and then check id=btn2 in both inside shadowroot as you did.

Expect that 
id=btn1 inside shadowroot => unchecked
id=btn2 inside  shadowroot => checked
id=btn1 outside  shadowroot => checked
id=btn2 outside  shadowroot => unchecked

Then we ensure that 1/ same group name of inside and outside shadowroot isn't interfere 2/ group is still working correctly if it's inside shadowroot.

```html
<ef-radio-button name="group" id="btn1" checked>1</ef-radio-button>
<ef-radio-button name="group" id="btn2">2</ef-radio-button>

<div>
   #shadowRoot
      <ef-radio-button name="group" id="btn1" checked>1</ef-radio-button>
     <ef-radio-button name="group" id="btn2">2</ef-radio-button>
</div>
```